src/main/java/training/RobotType.java

```diff
@@ -0,0 +1,6 @@
+package training;
+
+public enum RobotType {
+    SIMULATION,
+    PREVIOUSLY_SYNCOPA;
```

Comment:
```suggestion
    SYNCOPA;
```
iots the same syncopa u tus tus